analyses/focal-cn-file-preparation/seg-to-gene-mapping-atrt.Rmd

```diff
@@ -0,0 +1,64 @@
+---
+title: "Segment to gene mapping in ATRT"
+output: html_notebook
+author: J. Taroni for ALSF CCDL
+date: 2019
+---
+
+From [this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/182#discussion_r340587968) on #182:
+
+> may advise starting with ATRT's and SMARCB1 deletions
+
+Here, I'm looking at the output of `01-prepare-cn-file.R` to see if we recover that information in the segment to gene mapping.
+We're going to get the data from a pull request/branch that is currently open.
+This will get whatever the current files are for that branch.
+
+```{sh}
```

Comment:
I don't know if `results='hide'` works in shell blocks, but if not, maybe throw a `--quiet` on these?

```suggestion
{sh, results = 'hide'}
```

Reply:
Let's try it!

Reply:
Doesn't look that way, will do `--quiet`

Reply:
😞